### PR TITLE
Introduce Istio Sidecars

### PR DESCRIPTION
**Pull request by Lyczeq**
**Description**

Changes proposed in this pull request:

- Add create, list and details views
- Add translations
- Add tests
- Clean up the code and create reusable components

**Related issue(s)**
Closes: #946

### CODE REVIEW

File: core-ui/src/components/Predefined/Details/Sidecars.details.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  return (
+    <ul>
+      {hosts?.map((host, i) => (
+        <li key={i}>{host}</li>
+      ))}
+    </ul>
+  );
+};
+
+const Port = ({ port }) => {
```
Comment (akucharska) [inline core-ui/src/components/Predefined/Details/Sidecars.details.js:38]: Displaying port details does look a little bit funky. Can we make it more readable
Screen Shot 2022-02-16 at 08 05 57
especially when there might be multiple egress objects (like in sample Sidecar in ak namespace)

**Review by akucharska — COMMENTED** [review]
Consider if we can make ingress/egress prettier... maybe something like in pod containers? 
Screen Shot 2022-02-16 at 08 14 44

**Review by akucharska — DISMISSED** [review]

**Review by grego952 — APPROVED** [review]

**Pull request merged**